make/acceptance-tests.mk

```diff
@@ -11,6 +11,53 @@ else
 TEST_ACCEPTANCE_TAGS_ARG ?= --tags="~@disabled"
 endif
 
+ACCEPTANCE_TEST_TARGETS := test-acceptance test-acceptance-dr test-acceptance-x test-acceptance-wip test-acceptance-wip-x
+
+$(ACCEPTANCE_TEST_TARGETS): ensure-agentsvc-image ensure-agentapp-image ensure-controller-image
+
+ifeq ($(origin PRIMAZA_CONTROLLER_IMAGE_REF), undefined)
+export PRIMAZA_CONTROLLER_IMAGE_REF = primaza-controller:latest
+endif
+ifeq ($(origin PRIMAZA_AGENTAPP_IMAGE_REF), undefined)
+export PRIMAZA_AGENTAPP_IMAGE_REF = agentapp:latest
+endif
+ifeq ($(origin PRIMAZA_AGENTSVC_IMAGE_REF), undefined)
+export PRIMAZA_AGENTSVC_IMAGE_REF ?= agentsvc:latest
+endif
```

Comment:
These will be the defaults you can use in docs

Reply:
Fixed in 112f4d2.